packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -911,3 +915,36 @@ export const makeLCA = (async (orch: Orchestrator): Promise<LocalAccount> => {
   return agoricChain.makeAccount();
 }) satisfies OrchestrationFlow;
 harden(makeLCA);
+
+export const withdraw = (async (
+  orch: Orchestrator,
+  ctx: PortfolioInstanceContext,
+  seat: ZCFSeat,
+  pKit: GuestInterface<PortfolioKit>,
+) => {
+  const pId = pKit.reader.getPortfolioId();
+  const traceP = makeTracer('withdraw').sub(`portfolio${pId}`);
+
+  // cast justified by proposal shape.
+  const proposal = seat.getProposal() as unknown as ProposalType['withdraw'];
+  const { Cash: amount } = proposal.want;
+
+  // XXX enhancement: let caller supply steps
+  const { stepsP, flowId } = pKit.manager.startFlow({
+    type: 'withdraw',
+    amount,
+  });
```

Comment:
`{ type: 'withdraw', amount }` doesn't provide sufficient detail to differentiate requests for monotonic withdraw vs. withdraw with rebalance, so I think I'll have the planner interpret it as dealer's choice.